src/RA_Dlg_Memory.cpp

```diff
@@ -1440,7 +1421,7 @@ void Dlg_Memory::RepopulateMemNotesFromFile()
     size_t nSize = 0;
 
     const auto& pGameContext = ra::services::ServiceLocator::Get<ra::data::GameContext>();
-    const auto nGameID = pGameContext.GameId();
+    const auto nGameID       = pGameContext.GameId();
```

Comment:
I think I'd like to disable `AlignConsecutiveAssignments` as well. It's nice when assigning constant values, but is less pleasing when aligning function results.

Reply:
That should be fine. It causes more problems than it solves with that option on.